### ISSUE TITLE
Error on installation; OSX 10.7.5

### ISSUE THREAD

**Issue by scashin133** [bear-metal/tunemygc#20]
On bundle install, the install fails when installing tunemygc with the following stack trace:

```
checking for RUBY_INTERNAL_EVENT_GC_END_SWEEP... yes
creating Makefile

make "DESTDIR=" clean

make "DESTDIR="
compiling getRSS.c
getRSS.c: In function ‘getCurrentRSS’:
getRSS.c:97: error: storage size of ‘info’ isn’t known
getRSS.c:98: error: ‘MACH_TASK_BASIC_INFO_COUNT’ undeclared (first use in this function)
getRSS.c:98: error: (Each undeclared identifier is reported only once
getRSS.c:98: error: for each function it appears in.)
getRSS.c:99: error: ‘MACH_TASK_BASIC_INFO’ undeclared (first use in this function)
make: *** [getRSS.o] Error 1

make failed, exit code 2
```

**Comment by methodmissing** [comment]
Thanks for flagging, investigating

**Comment by methodmissing** [comment]
@scashin133 on an old OS X release perhaps?

**Comment by scashin133** [comment]
Yes, 10.7.5

On Monday, April 20, 2015, Lourens Naudé <notifications@github.com> wrote:

> @scashin133 <https://github.com/scashin133> on an old OS X release
> perhaps?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bear-metal/tunemygc/issues/20#issuecomment-94390570>.
>


-- 
http://about.me/seancashin/bio

**Comment by methodmissing** [comment]
Pushed a probable fix via https://github.com/bear-metal/tunemygc/commit/9652a759ab4eba7caa7a333383ebd7461e3040cf and release 1.0.57 . Mind taking it for a spin?

**Comment by scashin133** [comment]
I'll ask the engineer with the old machine to try installing this version.
Thanks for working so quickly!

On Monday, April 20, 2015, Lourens Naudé <notifications@github.com> wrote:

> Pushed a probable fix via 9652a75
> <https://github.com/bear-metal/tunemygc/commit/9652a759ab4eba7caa7a333383ebd7461e3040cf>
> and release 1.0.57 . Mind taking it for a spin?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bear-metal/tunemygc/issues/20#issuecomment-94502385>.
>


-- 
http://about.me/seancashin/bio
